packages/fluent/scss/listview/_layout.scss

```diff
@@ -0,0 +1,143 @@
+@use "../core/mixins" as *;
+@use "_variables.scss" as *;
+
+@mixin kendo-listview--layout() {
+
+    // Listview
+    .k-listview {
+        margin: 0;
+        padding: 0;
+        border-width: 0;
+        border-style: solid;
+        box-sizing: border-box;
+        outline: 0;
+        font-family: var( --kendo-listview-font-family, $kendo-listview-font-family );
+        font-size: var( --kendo-listview-font-size, $kendo-listview-font-size );
+        line-height: var( --kendo-listview-line-height, $kendo-listview-line-height );
+        flex-flow: column nowrap;
+        position: relative;
+        -webkit-touch-callout: none;
+        -webkit-tap-highlight-color: transparent;
+    }
+    .k-listview-bordered {
+        border-width: var( --kendo-listview-border-width, $kendo-listview-border-width );
+    }
+
+
+    // Listview content
+    .k-listview-content {
+        margin: 0;
+        padding: 0;
+        outline: 0;
+        flex-grow: 1;
+        position: relative;
+        overflow: auto;
+
+        &::after {
+            height: 0;
+            clear: both;
+            display: block;
+        }
+    }
+    .k-listview-content.k-d-flex,
+    .k-listview-content.k-d-grid {
+        &::after { display: none; }
+    }
+    .k-listview-content.k-d-grid {
+        grid-gap: var( --kendo-listview-grid-gap, $kendo-listview-grid-gap );
+    }
+    .k-listview-content-padding-rectangle {
+        padding: var( --kendo-listview-padding-y, $kendo-listview-padding-y ) var( --$kendo-listview-padding-x, kendo-listview-padding-x );
+    }
+    .k-listview-content-padding-square {
+        padding: var( --kendo-listview-padding-y, $kendo-listview-padding-y );
+    }
+
+
+    // Listview item
+    .k-listview-item {
+        padding: 0;
+        border-width: 0;
+        outline: 0;
+        border-style: solid;
+        box-sizing: border-box;
+        flex-shrink: 0;
+    }
+    .k-listview-item-padding-rectangle {
+        padding: var( --kendo-listview-item-padding-y, $kendo-listview-item-padding-y ) var( --$kendo-listview-item-padding-x, kendo-listview-item-padding-x );
+    }
+    .k-listview-item-padding-rectangle .k-listview-link {
+        margin: (-$kendo-listview-item-padding-y) (-$kendo-listview-item-padding-x);
+        padding: var( --kendo-listview-item-padding-y, $kendo-listview-item-padding-y ) var( --$kendo-listview-item-padding-x, kendo-listview-item-padding-x );
+    }
+    .k-listview-item-padding-square {
+        padding: var( --kendo-listview-item-padding-y, $kendo-listview-item-padding-y );
+    }
+    .k-listview-item-padding-square .k-listview-link {
+        margin: (-$kendo-listview-item-padding-y);
+        padding: var( --kendo-listview-item-padding-y, $kendo-listview-item-padding-y );
```

Comment:
use `margin-inline`, `margin-block` and `padding-inline`, `padding-block` for consistency. see previous comments.